src/code_FBE10.c

```diff
@@ -67,7 +67,27 @@ ApiStatus func_802D7B74(ScriptInstance* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "code_FBE10", func_802D7BA4);
 
-INCLUDE_ASM(s32, "code_FBE10", func_802D7CF8);
+ApiStatus func_802D7CF8(ScriptInstance* script, s32 isInitialCall) {
```

Comment:
Can call this `Spawn802D9D50`; it's equivalent to the script `spawn D_802D9D50` except variables 7..=16 aren't copied.